app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/adapter/RecentSearchesAdapter.kt

```diff
@@ -65,7 +64,7 @@ class RecentSearchesAdapter(
                         val room = RoomFinderRoom.fromRecent(recentSearch)
                         titleText.text = room.formattedAddress
                     } catch (exception: Exception) {
-                        titleText.setText(R.string.not_available_search)
+                        onRemove(recentSearch)
```

Comment:
I cannot reproduce to get an error from `fromRecent`. Is there a way this can fail that I am missing?
Can we just remove the try catch?